src/views/blocks/tickets/extra-price.php

```diff
@@ -8,18 +8,24 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @version 4.11.0
+ * @since   4.9
  *
+ * @version 4.11.0
```

Comment:
Any time we make a view template change, we have to increment to `@version TBD` and update `@since TBD What changed`

Reply:
will do